app/TestTCP/testenv.hpp

```diff
@@ -44,6 +44,11 @@ class TestEnv1 : public ::testing::Test
 
 	virtual void SetUp()
 	{
+		if(getenv("RANDOM_SEED") == NULL)
+			setenv("RANDOM_SEED", std::to_string(time(0)).c_str(), true);
+		printf("[RANDOM_SEED : %s]\n", getenv("RANDOM_SEED");
```

Comment:
Seems that there exists the typo here which makes compilation failed.
Need one more closing parentheses at the end.

`printf("[RANDOM_SEED : %s]\n", getenv("RANDOM_SEED"));`
@mdsnins Could you modify it in hotfix?